molgenis_documentation/user_documentation.md

```diff
@@ -161,25 +125,15 @@ The fastest and easiest way to get MOLGENIS running on a machine, is using our c
 <!--
 TODO: Get this project to the molgenis repository, so I do not have to link to the Github of fleur
```

Comment:
I've just transferred ownership to Molgenis so it's moved to https://github.com/molgenis/molgenis-cargo